x/group/internal/math/dec_test.go

```diff
@@ -291,7 +291,7 @@ func floatDecimalPlaces(t *rapid.T, f float64) uint32 { //nolint:unused
 	// Subtract exponent from base and check if negative
 	if res := basePlaces - exp; res <= 0 {
 		return 0
-	} else { //nolint:revive
+	} else { //nolint:revive // we're declaring the variable res in the if statement, so we won't be able to use it outside of the if statement
```

Comment:
I would just break the if statement.

res := foo
if condition {
 return 0
}

return res

Reply:
that was top ser